src/malli/util.cljc

```diff
@@ -182,6 +182,35 @@
          (swap! state conj path) nil)))
     @state))
 
+(defn data-explainer
+  "Like `m/explainer` but output is pure clojure data. Schema objects have been replaced with their m/form.
+   Useful when you need to serialise errrors."
+  ([?schema]
+   (data-explainer ?schema nil))
+  ([?schema options]
+   (let [explainer' (m/explainer ?schema options)
+         schema->data (fn [s]
```

Comment:
side-note: it's always a `Schema`, so `schema->data` could be replaced just with `m/form`.